.github/workflows/on-v-tag-release.yaml

```
@@ -9,6 +9,7 @@ permissions:
 
 jobs:
   release:
-    uses: harmony-labs/workflow-rust-release/.github/workflows/workflow.yaml@a042c53c336e636cada49d5d352ebe5f9b5ff452 # v2.0.2
+    uses: harmony-labs/workflows-rust/.github/workflows/release.yaml@v0.2.2
     with:
-      binary_name: ${{ github.event.repository.name }}
\ No newline at end of file
+      binary_name: ${{ github.event.repository.name }}
+      build_args: "--release --features vendored"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)